src/build.sh

```diff
@@ -94,6 +98,16 @@ function kernel_build()
 
   command="make -j$parallel_cores ARCH=$arch $cross_compile"
 
+  if [ -n "$pkg" ]; then
+    revision=$(cat .version) # Has to be gathered before building
```

Comment:
What does this version mean? Does the kernel generate this? Maybe add a top comment that describes this version file and why we need it.